services/workflows-service/src/business/schemas.ts

```diff
@@ -0,0 +1,63 @@
+import z from 'zod';
+import { zodBuilder } from '@ballerine/common';
+import { Prisma } from '@prisma/client';
+import { zApprovalStateEnum } from '@/filter/dtos/temp-zod-schemas';
+
+import { InputJsonValueSchema, RecordAnySchema } from '@/common/schemas';
+
+export const BaseBusinessRelationships = zodBuilder<
+  Pick<Prisma.BusinessCreateInput, 'workflowRuntimeData' | 'endUsers' | 'endUsersOnBusinesses'>
+>()(
+  z.object({
+    workflowRuntimeData: RecordAnySchema.optional(),
+    endUsers: z.array(RecordAnySchema).optional(),
+    endUsersOnBusinesses: z.array(RecordAnySchema).optional(),
+  }),
+);
+
+export const BaseBusinessCreateInputSchema = zodBuilder<
+  Omit<Prisma.BusinessCreateInput, 'project' | 'endUsers' | 'endUsersOnBusinesses'>
+>()(
+  z.object({
+    id: z.string().cuid().optional(),
+    correlationId: z.string().optional(),
```

Comment:
@MatanYadaev Did we say its gonna be required field?

Reply:
We concluded it, yes, but who's going to refactor it? :)

Reply:
Out of scope for my task. I'll update this schema myself once the field is no longer optional.